v3/deprecated/20250221-v3-cow-swap-fee-burner/readme.md

```diff
@@ -1,3 +1,8 @@
+> ⚠️ **DEPRECATED: do not use** ⚠️
+>
+> This version was superseded by [`CowSwapFeeBurner` V2](../../tasks/20250530-v3-cow-swap-fee-burner-v2/).
```

Comment:
Let's add a small explanation about why this one was deprecated (issues when repeating orders for the same token, access improvements, etc).